### PR TITLE
correct the private key path in the dev doc

### PR DESCRIPTION
**Pull request by mwrock**
Signed-off-by: mwrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: BUILDER_DEV.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This document outlines the steps to start and run a Builder environment for deve
 1. The sample commands below use the 'httpie' tool. Install it if not present on your system (https://github.com/jkbrzt/httpie).
 1. A recent version of the habitat cli should also be installed in your dev environment (https://www.habitat.sh/docs/get-habitat/)
 1. Ensure that `sudo -E` functions correctly. Some OSes don't respect the -E flag. If this is the case, remove the `secure_path` section of your sudoers file.
-1. Copy `habitat-builder-dev.2017-10-02.private-key.pem` from 1Password into `.secrets/builder-github-app.pem`
+1. Copy `habitat-builder-dev.2017-10-02.private-key.pem` from 1Password into `/src/.secrets/builder-github-app.pem`
```
Comment (chefsalim) [inline BUILDER_DEV.md]: Alternatively I think we can just set the builder-api config file to point to this - I'm testing that out

Comment (chefsalim) [reply BUILDER_DEV.md]: Also there are other changes to the builder api config file below - we need to remove the other github client id and secret config.

Comment (mwrock) [reply BUILDER_DEV.md]: there are several services that need to point here. `/src/.secrets/builder-github-app.pem` is the github config's default.

Comment (mwrock) [reply BUILDER_DEV.md]: We actually still need the other GH config properties.

**Comment by reset** [comment]
I wrote this with the idea that you'd be outside the container but if this is more clear, let's do it

**Pull request merged**